### DISCUSSION TITLE
[Kavita] Start LibraryWatcher task after Kavita startup

### DISCUSSION THREAD

**Discussion by therobbiedavis** [Kareadita/Kavita#3081] [Ideas]
### Idea Description

Currently if folder watching is enabled, Kavita is inaccessible as it boots up. This is not ideal in general but especially so for large libraries.

I propose maybe figuring out a way to enable folder watching on it's own thread that way Kavita can be used concurrently while the library watcher is starting to watch everything.

Look at my latest startup: over 40 minutes and still going.
![image](https://github.com/user-attachments/assets/56f53ac8-67fe-47e4-9206-4d152f14d31f)

### Idea Category

Performance Improvement

### Duration of Using Kavita

Multiple years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This was delivered in v0.8.2.6 and will be released in v0.8.3.
